frontend/components/service/comboForm.js

```diff
@@ -1,347 +1,372 @@
+                        <Grid item xs={8}>
+                            <h4>{item.name}</h4>
+                        </Grid>
+                        <Grid item xs={8}>
+                            <h4>{item.service_code}</h4>
+                        </Grid>
+                    </Grid>
+                </div>
+                <div>
+                    <div className={styled.flexAlignContainer}>
+                        <FormControl
+                            sx={{
+                                marginX: '10px',
+                            }}
+                        >
+                            <Select
+                                data-cy="patientSort"
+                                value={0}
+                                defaultValue=""
+                                // onChange={}
+                                sx={{
+                                    minWidth: '100px',
+                                }}
+                            >
+                                {!_isEmpty(item.durations_prices) ? item.durations_prices.map((durationPrice, index) => (
+                                    <MenuItem value={index} key={index}
+                                              onClick={() => changeDurationOfService(item.service_code, durationPrice, serviceCheckList)}>
+                                        H {durationPrice.duration} - ${durationPrice.price}
+                                    </MenuItem>
+                                )) : null}
+                            </Select>
+                        </FormControl>
+                        {isEdit ? (
+                            <Checkbox
+                                key={item.id}
+                                value={item}
+                                checked={
+                                    serviceCheckList ? serviceCheckList.includes(item) : false
+                                }
+                                onChange={(event) => {
+                                    handleServiceCheck(event.target.checked, item);
+                                }}
+                            />
+                        ) : (
+                            <IconButton onClick={() => removeService(item)}>
+                                <Close/>
+                            </IconButton>
+                        )}
+                    </div>
+                </div>
+            </div>
+        </Card>
+    );
 };
 const EditCombo = (props) => {
-  const {
-    editDialog,
-    setEditDialog,
-    serviceCheckListAddable,
-    handleServiceCheck,
-    removeService,
-    handleSave,
-  } = props;
-  const [serviceToAdd, setServiceToAdd] = useState([]);
-  const handleAddRemoveService = (val, item) => {
-    if (val) {
-      setServiceToAdd([...serviceToAdd, item]);
-    } else {
-      setServiceToAdd(
-        serviceToAdd.filter((itemService) => itemService.id != item.id)
-      );
-    }
-    // console.log(serviceCheckList)
```

Comment:
is the logger implemented? @yuxiang-zhang

Reply:
No, it is for my debugging, I forget to remove it. Good catch.